public/js/enform_submit.js

```diff
@@ -53,7 +53,7 @@ const p4_enform_frontend = (function ($) {
 
   enform.validateEmail = function(email) {
     // Reference: https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript
-    const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    const re = /^(([^<Fragment>()[\]\\.,;:\s@"]+(\.[^<Fragment>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Probably also because of a general find and replace?

Reply:
Indeed! 🤦 Solved